### DISCUSSION TITLE
Increase awareness of Pro version in Rectangle

### DISCUSSION THREAD

**Discussion by HJGreen** [rxhanson/RectanglePro-Community#555] [General]
Hi Ryan,

I've been a long-time user of Rectangle, and had no idea there was a Pro version available. After trying Rectangle Pro this morning I purchased straight away.

I wish I'd known there was a Pro version sooner - I only found out it existed from a random post on Twitter!

I appreciate it may be a concious decision not to push paid versions on your users, but might I suggest adding a menu item or link in the settings screens of Rectangle to the Rectangle Pro website to help other happy users find it and support your work?

**Comment by rxhanson** [comment]
Thanks for purchasing and for the positive feedback! Indeed, I have thought about this a lot and this is intentional. I could write a lot about this decision, but in short: I like Rectangle better without it there, even if it means less awareness of something else that I've spent a lot of time on.

**Reply by HJGreen** [reply]
I respect that. Thank you!
